### PR TITLE
Update zio-kafka version for quick start

### PR DESCRIPTION
**Pull request by GrigoriiBerezin**

### CODE REVIEW

**Comment by GrigoriiBerezin** [comment]
Was there any reason to have an internal module inside a Kafka module?

File: zio-quickstart-kafka/build.sbt [hunk 1]
```diff
@@ -1,10 +1,6 @@
-val `zio-quickstart-kafka` =
-  project.settings(
-    stdSettings(),
-    enableZIO(enableStreaming = true)
-  )
+scalaVersion := "2.13.8"
```
Comment (guizmaii) [inline zio-quickstart-kafka/build.sbt:1]:
```suggestion
scalaVersion := "2.13.13"
```

File: zio-quickstart-kafka/build.sbt [hunk 2]
```diff
@@ -1,10 +1,6 @@
-val `zio-quickstart-kafka` =
-  project.settings(
-    stdSettings(),
-    enableZIO(enableStreaming = true)
-  )
```
Comment (guizmaii) [inline zio-quickstart-kafka/build.sbt:5]: Why removing this? 🤔

Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/build.sbt:5]: I don't understand for what purpose it was. Is it required for something? :)

Comment (guizmaii) [reply zio-quickstart-kafka/build.sbt:5]: It probably declares the project itself (or its main module), no? Haven't looked at how the project is organised

Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/build.sbt:5]: Ok, will look at it, I thought declaration of module is declared in root sbt file

File: zio-quickstart-kafka/src/main/scala/dev/zio/quickstart/StreamingKafkaApp.scala [hunk 3]
```diff
@@ -31,20 +31,25 @@ object StreamingKafkaApp extends ZIOAppDefault {
...
         .repeatZIO(Clock.currentDateTime)
         .schedule(Schedule.spaced(1.second))
-        .map(time => new ProducerRecord(KAFKA_TOPIC, time.getMinute, s"$time -- Hello, World!"))
+        .map(time =>
+          new ProducerRecord(
+            KAFKA_TOPIC,
+            time.getMinute,
+            s"$time -- Hello, World!"
+          )
+        )
```
Comment (erikvanoosten) [inline zio-quickstart-kafka/src/main/scala/dev/zio/quickstart/StreamingKafkaApp.scala:40]: Usually, we use `{}` for a multi-line argument:
```suggestion
        .map { time =>
          new ProducerRecord(
            KAFKA_TOPIC,
            time.getMinute,
            s"$time -- Hello, World!"
          )
        }
```

File: zio-quickstart-kafka/build.sbt (hunk 2 above)
Comment (erikvanoosten) [reply zio-quickstart-kafka/build.sbt:5]: I interpret the text of the readme that each directory in this repo should be usable as a standalone project.

Anyways, none of the other projects have a similar setting. So removing this is probably fine.

**Review by erikvanoosten — APPROVED** [review]

File: zio-quickstart-kafka/build.sbt (hunk 2 above)
Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/build.sbt:5]: Yes, I've checked it - it just create inner project with the same name and empty target

File: zio-quickstart-kafka/build.sbt (hunk 1 above)
Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/build.sbt]: Just wanted to save the same versions, but ok, changed it

File: zio-quickstart-kafka/src/main/scala/dev/zio/quickstart/StreamingKafkaApp.scala (hunk 3 above)
Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/src/main/scala/dev/zio/quickstart/StreamingKafkaApp.scala]: Thanks, fixed

File: zio-quickstart-kafka/build.sbt (hunk 2 above)
Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/build.sbt:5]: So let's just remove it

Comment (GrigoriiBerezin) [reply zio-quickstart-kafka/build.sbt]: Noticed that without inner module I cannot run classes with kafka, so return it with some refactor

**Comment by GrigoriiBerezin** [comment]
Is it ok or should I make some improvements?

**Review by guizmaii — APPROVED** [review]

**Comment by erikvanoosten** [comment]
This is already a huge improvement!

Still, there are always more things to fix (since you're asking 😄 ):

- The `enableZIO(enableStreaming = true)` setting in `build.sbt` is not present in normal sbt projects. What does it do, can we replace it with something more standard.
- The consumer uses `tap` which destroys the chunking structure. In almost every application this will hurt performance a lot.

Regardless, I think this is already nice to merge! (@guizmaii who has merge rights in this project, you?)

**Comment by guizmaii** [comment]
> Regardless, I think this is already nice to merge! (@guizmaii who has merge rights in this project, you?)

I don't but I can ask to get the rights

**Comment by GrigoriiBerezin** [comment]
> This is already a huge improvement!
> 
> Still, there are always more things to fix (since you're asking 😄 ):
> 
> * The `enableZIO(enableStreaming = true)` setting in `build.sbt` is not present in normal sbt projects. What does it do, can we replace it with something more standard?
> * The consumer uses `tap` which destroys the chunking structure. In almost every application this will hurt performance a lot. Perhaps replace it with `mapChunks`, or just put a comment on it? 🤷
> 
> Regardless, I think this is already nice to merge! (@guizmaii who has merge rights in this project, you?)

Thanks, will do it in next MRs, I have some more quickstarts for repo :)

**Comment by GrigoriiBerezin** [comment]
@khajavi has the rights I guess

**Pull request merged**